cli/command/environment_test.go

```diff
@@ -0,0 +1,419 @@
+package command
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/quintilesims/layer0/api/job"
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/urfave/cli"
+)
+
+func TestEnvironmentCreate_userInputErrors(t *testing.T) {
+	base, ctrl := newTestCommand(t)
+	defer ctrl.Finish()
+
+	command := NewEnvironmentCommand(base.Command())
+
+	contexts := map[string]*cli.Context{
+		"Missing NAME arg": NewContext(t, nil, nil),
+		"OS missing":       NewContext(t, Args{"env_name"}, nil),
+		"Count negative": NewContext(t, Args{"env_name"},
+			Flags{
+				"os":        "linux",
```

Comment:
if my comment above is true, then we don't need this for this test case

Reply:
see above.